src/app/signature-preview/signature-preview.component.scss

```diff
@@ -19,3 +19,7 @@ p a, ul a, table a {
   border-bottom: none !important;
   background-image: none !important;
 }
+
+#indigenousImage{
```

Comment:
Any particular reason an ID was used for this? If not, can we change it to a class please